webpack.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     entry: {
         all: './openlibrary/plugins/openlibrary/js/index.js',
         vue: './openlibrary/plugins/openlibrary/js/vue.js',
+        patnerLib: './openlibrary/plugins/openlibrary/js/patner_ol_lib.js'
```

Comment:
Good catch @jimman2003 !
```suggestion
        partnerLib: './openlibrary/plugins/openlibrary/js/partner_ol_lib.js'
```

Reply:
Great catch @jimman2003 ! Thanks for pointing this out. Fixed this @ https://github.com/internetarchive/openlibrary/pull/5479/commits/e01de275a35bc1dca505748bdfc35ea6f9d6ba4b